xrpl/models/currencies/issued_currency.py

```diff
@@ -53,3 +54,23 @@ def _get_errors(self: IssuedCurrency) -> Dict[str, str]:
         elif not _is_valid_currency(self.currency):
             errors["currency"] = f"Invalid currency {self.currency}"
         return errors
+
+    def to_amount(
+        self: IssuedCurrency, value: str
+    ) -> xrpl.models.amounts.IssuedCurrencyAmount:
+        """
+        Converts an IssuedCurrency to an IssuedCurrencyAmount.
+
+        Args:
+            value: The amount of issued currency in the IssuedCurrencyAmount.
+
+        Returns:
+            An IssuedCurrencyAmount that represents the issued currency and the
+                provided value.
+        """
+        return cast(
+            xrpl.models.amounts.IssuedCurrencyAmount,
+            xrpl.models.amounts.IssuedCurrencyAmount.from_dict(
+                {**self.to_dict(), "value": value}
```

Comment:
Since it's just two fields (`Account` and `currency`) I feel like it would be better to just explicitly reconstruct a new `IssuedCurrencyAmount` to avoid any weirdness where adding a field to this object changes the amount. (But it's a style pref, this seems to work)

Reply:
Hahaha, you're way ahead of me!